LICENSE

```diff
@@ -1,21 +1 @@
-MIT License
 
-Copyright (c) 2023 Yuhong Sun, Chris Weaver
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
```

Comment:
Hey, just want to point out, please do not remove the LICENSE from your fork. This makes us sad 😞. The License in it's entirety must be distributed with any significant portions of the code.

Also you probably didn't intend to create these PRs so I'll just close them for you ok? :)